dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -199,6 +201,11 @@ internal Dictionary<string, object> ToDictionary()
                 toReturn["shrinkToFit"] = this.shrinkToFit;
             }
 
+            if (this.pageSize == null) 
+            { 
+                this.pageSize = new PageSize(); 
+            }
+            
             if (this.pageSize.Height != DefaultPageHeight || this.pageSize.Width != DefaultPageWidth)
```

Comment:
It can be optimized to avoid allocation: `if (this.pageSize != null && (this.pageSize.Height != DefaultPageHeight || this.pageSize.Width != DefaultPageWidth))`